### DISCUSSION TITLE
check multi authenticator ??

### DISCUSSION THREAD

**Discussion by CodingBDX** [symfony/symfony#49304] [Q&A]
### Description

hello:
i have website with multi users, i just want to show 'all users connected"???
i have make for register, i put one condition into my registercontroller, but i don't know, how to put in login or logout...
please help me

**Comment by wouterj** [comment]
Hi! Please use one of our community support channels to ask for help: https://symfony.com/support

This issue tracker is focused on tracking bugs and feature requests.

**Comment by lyrixx** [comment]
Hello, I don't understand what you want to achieve.
But you try to 
* read the documentation https://symfony.com/doc/current/security.html
* read the book to start with symfony: https://symfony.com/book
